plugins/SitesManager/API.php

```diff
@@ -429,13 +434,13 @@ protected function getNormalizedUrls($url)
     public function getSitesIdFromSiteUrl($url)
     {
         $url = $this->removeTrailingSlash($url);
-        list($url, $urlBis) = $this->getNormalizedUrls($url);
+        list($url, $urlBis, $urlTer, $urlQuater) = $this->getNormalizedUrls($url);
```

Comment:
See comment further below in `getAllSitesIdFromSiteUrl`. Maybe don't do a `list` and just take the array of URLs and pass it to `getAllSitesIdFromSiteUrl()`